packages/webapp/components/layouts/AccountLayout/EmailForm.tsx

```diff
@@ -55,7 +59,11 @@ function EmailForm({
           label={passwordProps?.label || 'Password'}
         />
       )}
-      <Button className="mt-6 ml-auto w-fit bg-theme-color-cabbage btn-quaternary">
+      <Button
+        className="mt-6 ml-auto w-fit text-theme-label-primary"
```

Comment:
I don't think the text override is needed, but maybe also tripple check with design they ok with it.

Reply:
I checked and the result is we should keep the default: https://dailydotdev.slack.com/archives/C0381BQUEN9/p1702475962350259?thread_ts=1702475717.398979&cid=C0381BQUEN9

Reply:
Thanks for checking, I thought so :)